src/plays/typeracer/Readme.md

```diff
@@ -0,0 +1,27 @@
+# TypeRacer
+

```

Comment:
Please record a video on stackstream and add the video link by editing your play from the localhost.